api/views/canteen.py

```diff
@@ -1133,8 +1133,9 @@ def post(self, request, canteen_pk):
                     import_source=f"Cuisine centrale : {canteen.siret}",
                     production_type=Canteen.ProductionType.ON_SITE_CENTRAL,
                 )
-            for manager in canteen.managers.all():
-                satellite.managers.add(manager)
+            if created or satellite.managers.count() == 0:
```

Comment:
This `if` might seem redundant since all created satellites won't have managers, but I think in those cases it allows us not to query again the DB